commons/AndroidManifest.xml

```diff
@@ -1,7 +1,7 @@
 <manifest xmlns:android="http://schemas.android.com/apk/res/android"
     package="fr.free.nrw.commons"
-    android:versionCode="18"
-    android:versionName="1.3" >
+    android:versionCode="19"
+    android:versionName="1.31" >
```

Comment:
Oh, okay! I had no idea, I thought it worked like decimals. So I'll change it to 1.4, then in the future if we release a version with a big change like category suggestions we can skip to 2.0?